__tests__/app.test.js

```diff
@@ -178,4 +179,47 @@ describe("GET/api/articles/:article_id", () => {
         expect(res.body.message).toBe("Bad Request");
       });
   });
+  test.only("get 201 response with a new posted comment", () => {
+    const newComment = {
+      userName: "lurker",
+      body: "Good work on the article",
+    };
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+
+        console.log(response.body)
+        expect(response.body.message).toBe("Good work on the article");
```

Comment:
I would check the whole comment object here and not just the message! Have a look at the other properties